frontend/packages/core/src/Assests/global.ts

```diff
@@ -0,0 +1,33 @@
+import styled from "@emotion/styled";
+
+export const STYLE_MAP = {
+  xsmall: {
+    width: 18,
```

Comment:
really minor nit: can we extract these out to constants to reduce the duplication?

Reply:
updated, lmk if this isn't what you had in mind.

Reply:
just pushed up another change to reduce more code duplication. the style map now just has size instead of width/height since both values are the same.